client/src/components/hpaPanel.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Field from './field';
+
+const HpaPanel = ({spec}) => (
+    <>
+        {spec &&
+            <div>
+                <div className='contentPanel_header'>Container</div>
+                <div key="hpa" className='contentPanel'>
+                    {spec.minReplicas && (
```

Comment:
In this case "flasy" check may or may not be needed. Technically, if you left it out it would always show the "label" but the value would might be empty. This way it may not include the label at all. My gut feeling is that from a UX standpoint it might be less confusing to an end user if the field label exists with an empty value. The other way, it may not be clear to a user why that field is missing.

But, if you do feel strongly about the current approach I think you'd need to modify the check slightly. As it's written here, if `minReplicas === 0`, this field won't be displayed

Reply:
Ah, got you. I'm agree with you that we should still display it even if it's empty. I will remove these checks.